### ISSUE TITLE
Menu blinks on key commands even when they should be ignored

### ISSUE THREAD

**Issue by audionerd** [wonderunit/storyboarder#329]
Steps

- Type in a text box (like duration) and press a number key (which triggers the toolbar)

Expected

- Menu shouldn't blink when you press a number key that the UI is ignoring in favor of the text box

Actual

- Menu blinks

**Comment by setpixel** [comment]
Yeah - this is super annoying about how the menu accelerator works. It catches the accelerator no matter what. I don't know of a way to fix it.
